sponsorship_compassion/models/project_compassion.py

```diff
@@ -21,7 +21,8 @@ def _compute_sponsorships_count(self):
 
         for project in self:
             project.sponsorships_count = self.env['recurring.contract'].\
-                search_count([('child_id.project_id', '=', project.id)])
+                search_count([('child_id.project_id', '=', project.id),
+                              ('state', 'not in', ['cancelled', 'terminated'])
```

Comment:
You forgot to close one parenthesis. The build is failing because of this syntax error.